pkg/theiactl/commands/check.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"antrea.io/theia/pkg/theiactl/portforwarder"
+	sparkv1 "antrea.io/theia/third_party/sparkoperator/v1beta2"
+)
+
+// checkCmd represents the check command
+var checkCmd = &cobra.Command{
```

Comment:
Do you think should we categorize check.go/result.go/start.go and policyreco.go into one directory? I think we may have some new theiactl features in the future?